en/docs/reference/connectors/salesforce-connectors/sf-inbound-endpoint-example.md

```diff
@@ -2,27 +2,27 @@
+In this example, we can trigger the notifications to the Salesforce Inbound Endpoint via creating the `Platform events` or `PushTopic` methods. Please note that our example configurations are based on creating the `PushTopic` method. You can use the instructions given in the [sf-rest inbound endpoint configuration]({{base_path}}/reference/connectors/salesforce-connectors/sf-inbound-endpoint-configuration/) documentation.
 
 The following diagram illustrates all the required functionality of the Salesforce inbound operations that you are going to build. 
 
 For example, we are building an integrated example driven through the [Salesforce connector]({{base_path}}/reference/connectors/salesforce-connectors/sf-rest-connector-example/) and Salesforce Inbound Endpoint. The user calls the Salesforce REST API. It invokes the **create** sequence and creates a new account in Salesforce. Then, through the **retrieve** sequence, it displays all the existing account details to the user.
 
-Now that you have configured the Salesforce Inbound Endpoint, use the following Inbound Endpoint configuration to retrieve account details from your Salesforce account. The Salesforce inbound endpoint acts as a message receiver. You can inject that message into the mediation flow for getting the required output.
+Now that you have configured the Salesforce Inbound Endpoint, use the following Inbound Endpoint configuration to retrieve account details from your Salesforce account. The Salesforce inbound endpoint acts as a message receiver. You can inject that message into the mediation flow to get the required output.
 
 <a href="{{base_path}}/assets/img/integrate/connectors/salesforce-inboundep-example.png"><img src="{{base_path}}/assets/img/integrate/connectors/salesforce-inboundep-example.png" title="Salesforce Inbound Endpoint" alt="Salesforce Inbound Endpoint"/></a>
 
 ## Configure inbound endpoint using WSO2 Micro Integrator Visual Studio Code extension
 
 1. Follow the steps in [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up the Integration Project.
 
-2. First let's create a sequence to process the message. In this example for simplicity we will just log the message, but in a real world use case, this can be any type of message mediation. Select Micro Integrator and click on `+` in **Sequences** to create the sequence. Add a log and drop mediator to the sequence. The source view of the sequence will look like below.
+2. First let's create a sequence to process the message. In this example, for simplicity, we will just log the message, but in a real-world use case, this can be any type of message mediation. Select Micro Integrator and click on `+` in **Sequences** to create the sequence. Add a log and drop the mediator to the sequence. The source view of the sequence will look like below.
```

Comment:
```suggestion
2. First, let's create a sequence to process the message. In this example, for simplicity, you will only log the message, but in a real-world use case, this can be any type of message mediation. Select Micro Integrator and click on `+` in **Sequences** to create the sequence. Add a log and drop the mediator to the sequence. The source view of the sequence will look like below.
```